tests/files/test-inventory

```diff
@@ -1,4 +1,2 @@
-test-bastion ansible_ssh_host=127.0.0.1 ansible_ssh_user=ubuntu
```

Comment:
These variables should migrate down to the `test-bastion` line you've left remaining, or we'll lose that data? or is it defined somewhere else?

Also, why connect over ssh instead of using the local connection method? (not leaving negative review)